### PR TITLE
Feature/docker compose file

### PR DESCRIPTION
**Pull request by vilherda**
# Pull Request Template

## PR Checklist

- [ ] I have run `npm test` locally and all tests are passing.
- [ ] I have added/updated tests for any new behavior.
- [ ] If this is a significant change, an issue has already been created where the problem / solution was discussed: [N/A, or add link to issue here]

## PR Description

initial version of the file for development purposes

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    #     condition: any
+    #     delay: 10s
+    #     max_attempts: 3
+    #     window: 60s
+    # OK
+    entrypoint: "/bin/bash -c 'sleep 40s && tini -p SIGTERM -- /docker-entrypoint.sh mongo-express'"
+    # KO:
+    # entrypoint: "/bin/bash -c 'sleep 40s' && tini -p SIGTERM -- /docker-entrypoint.sh mongo-express"
+    # OK
+    # entrypoint: "tini -p SIGTERM -- /docker-entrypoint.sh mongo-express"
```
Comment (ericzon) [inline docker-compose.yml]: Clean unused commands or if you think it worths it then put a comment describing the reason

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      - ${PWD}/mongo_data/data:/data/db
+      - ${PWD}/mongo_data/init:/docker-entrypoint-initdb.d/
+      # - ${PWD}/mongo_data/configdb:/data/configdb
+    expose:
+      - 27017
+    networks:
+      - nlpjs
+    healthcheck:
+      test: ["CMD", "mongostat", "-u", "myadmin", "-p", "myadmin", "--authenticationDatabase=admin", "--json", "-n", "1"]
+      # test: ["CMD", "mongotop", "-u", "myadmin", "-p", "myadmin", "--authenticationDatabase=admin", "--json", "-n", "1"]
```
Comment (ericzon) [inline docker-compose.yml]: Remove it or add a comment explaining how it contributes

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    stop_signal: SIGTERM
+    healthcheck:
+      test: ["CMD", "wget", "--spider", "--quiet", "-T", "2", "-t", "1", "http://localhost:3000"]
+      interval: 30s
+      timeout: 10s
+      retries: 3
+      start_period: 30s
+    depends_on:
+      - mongo
+    # entrypoint: "/bin/bash -c 'npm cache clean --force && npm install && sleep 3s && node .'"
```
Comment (ericzon) [inline docker-compose.yml]: Clean

Comment (vilherda) [reply docker-compose.yml]: I don't clean the variant, I have explain it.

File: docker-compose.yml (hunk 2 above)
Comment (vilherda) [reply docker-compose.yml]: I don't clean the variant, I have explain it.

**Comment by ericzon** [comment]
@vilherda as this is an interesting feature for the end-user please, add some instructions in the README

**Pull request merged**